std_msgs/MultiArrayLayout layout
float32[] r11

float32[] r12

float32[] r21

float32[] r22